Adding new indirect selection pages (#4977)
fishtown-analytics/docs.getdbt.com: website/docs/docs/core/connect-data-platform/profiles.yml.md

Reviewer: I _think_ is moving to `dbt_project.yml` within https://github.com/dbt-labs/docs.getdbt.com/pull/4814

So we'll either need to merge that one first or merge this one first.

Then deal with any merge conflicts that come up.

Author: @dbeatty10 I think I might change the base branch to #4814 and deal with the merge conflicts now.

  [debug](/reference/global-configs/logs#log-level): <true | false>
  [version_check](/reference/global-configs/version-compatibility): <true | false>
  [fail_fast](/reference/global-configs/failing-fast): <true | false>

>   [indirect_selection](/reference/global-configs/indirect-selection): <eager | cautious | buildable | empty>

Author: We might want to move all of these flags into the project file

Reviewer: Good call!

It's covered in #4814 ([preview](https://docs-getdbt-com-git-jerco-flags-behavior-changes-dbt-labs.vercel.app/reference/dbt_project.yml)) with a link to `<global-configs>` like this:

<img width="578" alt="image" src="https://github.com/dbt-labs/docs.getdbt.com/assets/44704949/9b78de52-cac5-4fcb-93ac-f001c9bc1a41">

## User config

> You can set default values of global configs for all projects that you run using your local machine. Read the docs on [global configs](/reference/global-configs/about-global-configs) for details.

Reviewer 2: sugg: more inclusive word choice for less abled people

```suggestion
You can set default values of global configs for all projects that you run using your local machine. Refer to [About global configs](/reference/global-configs/about-global-configs) for details.
```
